medicines/web/src/services/documents-loader.ts

```diff
@@ -73,7 +73,7 @@ const convertDocumentResponseToDocument = ({
 }): IDocument => {
   return {
     activeSubstances: doc.activeSubstances,
-    context: doc.highlights,
+    context: doc.highlights?.join(' … ') || '',
```

Comment:
interested to know what this line does...

Reply:
`highlights` come back from Azure, the parts of the blobs where the search terms appear, and we stitch them together with a little `&hellip;` – I'd missed this previously